### DISCUSSION TITLE
Twitch videos

### DISCUSSION THREAD

**Discussion by AtomFry** [jmbannon/ytdl-sub#676] [Q&A]
This is working great for me to download YouTube channels. I can also use ytdlp to download the videos on a twitch channel. Is it possible to create a ytdl-sub config that can download the videos on a twitch channel or would that be a new feature request?

**Comment by jmbannon** [comment]
You can absolutely create a twitch config (or any other site supported by yt-dlp). I haven't tried ripping from twitch myself, so I'm not sure if they're compatible with the existing TV show presets. That's worth trying though.

I good starting point is to take the url you're interested in downloading, and run:
```
ytdl-sub view "<url>"
```

This will show you all the available source variable values for the first video it pulls. Docs: https://ytdl-sub.readthedocs.io/en/latest/usage.html#view-options

**Comment by juddiDaniel** [comment]
You can download twitch videos within seconds using [ZeTwitch](https://zetwitch.com/), you can check it and be sure to love the services
